media/css/mozorg/contribute/contribute-2015.less

```diff
@@ -1543,6 +1543,19 @@ textarea:focus {
     }
 }
 
+html[lang="ar"] .events-list {
```

Comment:
Instead of only <code>[lang='ar']</code> we can apply this to all RTL languages with the selector <code>[dir='rtl']</code>. We can also omit the 'html' part and just lead with the attribute selector. All the descendant rules can be nested under a single attribute selector and avoid some repetition:
<pre><code>[dir='rtl'] {
    .events-list {
        ...
    }

    .events-table {
        ...
    }
}</code></pre>